requirements.txt

```diff
@@ -8,3 +8,4 @@ MarkupSafe==1.0
 py-gfm==0.1.3
 pytz==2018.5
 requests==2.19.1
+configobj==5.0.6
```

Comment:
Do we need to introduce that dependency? Maybe Django has a builtin function for this?

Reply:
Didn't found (and looked a lot) the problem is that .py templates aren't really standard, usually it's INI, YAML, JSON etc'.
The one I found is for INI but with a little bit of hack it can be used to .py.
The alternative is to write something ourself but it will be ugly